lib/player.js

```diff
@@ -1,13 +1,42 @@
 const Mplayer = require('mplayer')
+const fs = require('fs')
+const m3u = require('m3u')
+const moment = require('moment')
 
 let mplayer
 let contents = []
 let isPlaying = false
 let playingContent
 
+const createNewPlaylistFile = (previousTriesNb = 0) => {
+  let playlistName = moment().format('YYYY-MM-DD')
+  if (previousTriesNb) playlistName += `-${previousTriesNb}`
+  // Try accessing a file with the planned playlist filename.
+  // If it exists, then find another name.
+  try {
+    fs.accessSync(`${playlistName}.m3u`, fs.F_OK)
+    return createNewPlaylistFile(previousTriesNb + 1)
+  } catch (e) {
+    return `${playlistName}.m3u`
+  }
+}
+const playlistName = createNewPlaylistFile()
+
 const getPlaylist = () => contents
 const setPlaylist = elements => (contents = [...elements])
 const addToPlaylist = content => contents.push(Object.assign({}, content))
+const savePlaylist = () => {
+  const m3uWriter = m3u.writer()
+  contents.forEach(content => m3uWriter.file(content.path))
+  fs.writeFileSync(playlistName, m3uWriter.toString(), { encoding: 'utf8' })
+}
+const loadPlaylist = (filepath, allContents) => {
+  const filePaths = fs.readFileSync(filepath, { encoding: 'utf8' })
+    .split('\n')
+    .filter(filePath => filePath[0] !== '#') // Comments in m3u files start with #
```

Comment:
`!filePath.startsWith('#')` looks prettier (in fact, a function not would look ever prettier but I may be infected)